roles/ceph-dashboard/tasks/configure_dashboard.yml

```diff
@@ -60,11 +60,13 @@
   shell: |
     {{ container_exec_cmd }} ceph config set mgr mgr/dashboard/server_port {{ dashboard_port }} || \
     {{ container_exec_cmd }} ceph config-key set mgr/dashboard/server_port {{ dashboard_port }}
+  changed_when: false
```

Comment:
if I remember correctly, this is because we want to support multiple ceph release here

Reply:
Yes but it's for pre nautilus release. Considering that the dashboard feature is only stable-4.0+ (so nautilus+) we should only have one command.
BTW this was already mentioned in the inital PR https://github.com/ceph/ceph-ansible/pull/3426#discussion_r280804697